src/typescript/4-types.md

```diff
@@ -201,7 +201,7 @@ In this exercise, we will learn to spot and correct basic type errors by
 fixing the type errors in `2a-fix-errors.ts`:
 
 ```typescript
-let isLoading: boolean = true;
+let isLoading = true;
```

Comment:
for this one, maybe we should just Type is as boolean, and not declare an initial value, I think that may be clearer to new reader.